src/components/ScrollDownArrow/styles.scss

```diff
@@ -1,32 +1,25 @@
 .arrow {
   cursor: pointer;
-  left: 50vw;
+  left: calc(50vw - 15px);
```

Comment:
arrow wasn't centered before on the page. we had left edge of the image in the middle rather than the center of the image. subtracted half of the width to fix this